recipes/libevent/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Needed to ensure our OpenSSL and
+# not the system one is used on OS X.
+export LIBRARY_PATH="${PREFIX}/lib"
+
+export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+
+# Set the fallback library environment variable.
+if [[ `uname` == 'Darwin' ]];
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+else
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+
+chmod +x ./autogen.sh
+
+./autogen.sh
+./configure --prefix="${PREFIX}"
+make
+
+#
+# Seems to hang on Mac builds. So have commented it for now.
+#
+#eval ${LIBRARY_SEARCH_VAR}="${PREFIX}/lib" make check
```

Comment:
So that's what the `LIBRARY_SEARCH_VAR` does as can be seen above. This is determined by OS platform. That being said, the tests just hang. So have commented them out for now. Will save this one for the feedstock.